src/Input/Touch/GestureDetector.cs

```diff
@@ -0,0 +1,640 @@
+﻿using System;
```

Comment:
`#region License` at the top of the file, and `#region Using Statements` for these lines here.